doc/source/ray-air/examples/pytorch_tabular_starter.py

```diff
@@ -33,27 +33,16 @@
 import numpy as np
 import pandas as pd
 
-from ray.data.preprocessors import BatchMapper, Chain
+from ray.data.preprocessors import BatchMapper, Chain, Tensorizer
```

Comment:
Is Tensorizer a common name? Would prefer something like Concatenate instead.

Reply:
fixed